ui/apps/platform/src/types/imageIntegration.proto.ts

```diff
@@ -45,8 +45,16 @@ export type ArtifactRegistryImageIntegration = {
 
 export type AzureImageIntegration = {
     type: 'azure';
+    azure: AzureConfig;
 } & BaseImageIntegration;
 
+export type AzureConfig = {
+    endpoint: string;
+    username: string; // scrub: always
```

Comment:
I'm not sure if it is relevant - this is not in sync with proto definition. We have `dependent` in proto and `aways` in TypeScript.

Reply:
fixed